web/client/utils/mapinfo/__tests__/wms-test.js

```diff
@@ -108,4 +110,38 @@ describe('mapinfo wms utils', () => {
             }
         );
     });
+    it('test intercept ogc error in wms if success', (done)=>{
+        mockAxios.onGet().reply(() => {
+            return [200, {
+                "type": "FeatureCollection",
+                "features": [],
+                "totalFeatures": "unknown",
+                "numberReturned": 0,
+                "timeStamp": "2023-09-22T08:50:30.808Z",
+                "crs": null
+            }];
+        });
+        wms
+            .getIdentifyFlow(undefined, "/", { features: [] })
+            .subscribe((response) => {
+                expect(response?.data?.features).toEqual([]);
+                done();
```

Comment:
remove this call so that we are sure it works because it triggers the error call back